packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -69,7 +69,7 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
     });
 
     it('opens the dropdown with space', () => {
-      browser.keys(['Escape', 'Space']);
+      browser.keys(['Tab', 'Escape', 'Space']);
```

Comment:
here also 'Tab' is not required to close dropdown. Please check for other test cases also.